root/plex_service.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-
-if [ "$#" -eq 1 ]; then
-  s6-svc "$1" /var/run/s6/services/plex
-else
-  echo "No argument supplied; must be -u, -d, or -r."
-fi
```

Comment:
Why is this removed?  This file is sometimes used by people who wish to stop the service inside a shell in the container without stopping the container.

Reply:
Re-added